spec/03-plugins/18-acl/02-access_spec.lua

```diff
@@ -477,6 +1065,17 @@ for _, strategy in helpers.each_strategy() do
         assert.res_status(200, res)
       end)
 
+      it("should work when one of the ACLs in the whitelist with authenticated groups", function()
+        local res = assert(proxy_client:send {
+          method  = "GET",
+          path    = "/request",
+          headers = {
+            ["Host"] = "acl7b.com"
+          }
+        })
```

Comment:
These calls could be written in one line:
```
local res = proxy_client:get("/request", { headers = { Host = "acl7b.com } })
```

Reply:
Every `:send` in whole `acl` test suite was now changed to use `:get/:post/:patch/:delete/:put`.